release/pkg/manager/calico/manager.go

```diff
@@ -203,6 +202,7 @@ type CalicoManager struct {
 	// image scanning configuration.
 	imageScanning       bool
 	imageScanningConfig imagescanner.Config
+	imageComponents map[string]registry.Component
```

Comment:
formatting looks off here - goimports?

Reply:
i missed adding it after i ran `make fix-changed`